firedrake/ml/__init__.py

```diff
@@ -1,2 +1 @@
-from firedrake.ml.backend_base import load_backend    # noqa: F401
-from firedrake.ml.pytorch import *                    # noqa: F401
+from firedrake.ml.pytorch import FiredrakeTorchOperator, torch_operator, from_ml_backend, to_ml_backend   # noqa: F401
```

Comment:
You don't want this line. It causes `from firedrake import ml` to import the PyTorch module. Now assume somebody implements a tensor flow backend. You wouldn't want the ml import to fail because of no PyTorch.